WalletWasabi.Fluent/Views/Shell/MainView.axaml

```diff
@@ -13,7 +13,7 @@
                     d:DesignWidth="1000" d:DesignHeight="1000"
                     x:CompileBindings="True">
 
-  <DockPanel IsEnabled="{Binding IsMainContentEnabled}">
+  <DockPanel IsEnabled="{Binding IsMainContentEnabled^}">
```

Comment:
What is this ^ ?

Reply:
That is Avalonia's cool feature, the [Binding to observables](https://docs.avaloniaui.net/docs/data-binding/binding-to-tasks-and-observables).

Instead of creating an observable, subscribing to it and storing the actual value in variable, we can just simply store the observable and bind to it.